package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "migrate": "node migrate.js up",
+    "migrate-update": "node migrate.js update",
```

Comment:
What's the difference between this and the regular `migrate` script above?

Reply:
@alexanmtz I added the migration script majorly to enhance working with an update of migrations in the development environment.

- The existing migration script will fail to apply a specific pending migration file if any of the migration files with timestamps before it could not migrate successfully.

- I encountered this with the create Plan migration which was totally unrelated to my label migration but was failing to migrate on my local environment hence preventing the label table from being migrated because it came before it.

- This script ensures it migrates individual pending migration files independently.

- This best used for development purposes as I wouldn't want to amend existing migration files. 

- Lastly, something to notice is that I updated the label-migration timestamps to come before the create-plan migration to prevent us from using this `migrate update` script on production and test.

will welcome any feedback on this approach.

Reply:
@Adekoreday great, let's add this command 👍